src/control/common/file_utils.go

```diff
@@ -284,3 +287,26 @@ func FindBinary(binName string) (string, error) {
 
 	return adjPath, nil
 }
+
+// GetFileOwner returns the username of user who owns the file at the given
+// path. Caller should check os.IsNotExist(errors.Cause(err)) to work out if
+// the call failed because the path doesn't exist or is a directory.
+func GetFileOwner(path string) (string, error) {
+	info, err := os.Stat(path)
+	if err != nil {
+		return "", err
+	}
+	if info.IsDir() {
+		return "", os.ErrNotExist
+	}
```

Comment:
This function is not general enough to belong in the common package, IMO. What if you want to get the owner of a directory? What if the path is to something other than a file or directory? What if you want to get the group?

Rather than overengineering this function to be more general (for as-yet-to-be-determined use cases), I suggest pulling it back into the bdev package and making it private.